src/woo_search/search_index/tests/test_search_api.py

```diff
@@ -1158,3 +1158,67 @@ def test_boost_recently_published_items(self):
         self.assertEqual(
             second["record"]["uuid"], "6dae9be7-4f93-4aad-b56a-10b683b16dcc"
         )
+
+    def test_dutch_analyzer(self):
+        """
+        Dutch analyzer test. Use a word which is present in the example of:
+        https://snowballstem.org/algorithms/dutch/stemmer.html
+
+        This way we have a clear reference point. for our test case we will use 'lichamelijk'.
+        So we are going to test that `lichamelijk`, `lichamelijke` and `lichamelijkheden` will result in a match.
+        """
+
+        index_document(
+            **IndexDocumentFactory.build(
+                uuid="828df354-b6dc-4693-815a-1b7d39b3bc95",
+                identifier="dutch-analyzer",
+                officiele_titel="Dutch analyzer",
+                omschrijving=""
+                "Een document om de Nederlandse analyzer te testen. "
+                "We doen dit door middel van een willekeurige stem (in ons geval "
+                "'lichamelijk') in deze tekst te vermelden, zodat we kunnen zoeken met"
+                "woorden die de zelfde stem bevatten.",
+            )
+        )
+
+        with self.subTest("lichamelijk"):
+            response = self.client.post(self.url, {"query": "lichamelijk"})
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+            data = response.json()
+
+            self.assertEqual(data["count"], 1)
+            self.assertFalse(data["previous"])
+            self.assertFalse(data["next"])
+            # test if results have the same length as the count
+            self.assertEqual(len(data["results"]), 1)
+            self.assertEqual(data["results"][0]["type"], "document")
```

Comment:
```suggestion
            self.assertEqual(data["count"], 1)
```
the outcome of this test really is there is a hit or there is none - the assertiosn about next/previous and document type etc. are all irrelevant